assets/javascripts/discourse/components/modal/user-notes.hbs

```diff
@@ -22,7 +26,7 @@
           {{#if n.can_delete}}
             <span class="controls">
               <DButton
-                @action={{action "removeNote" n}}
+                @action={{(fn this.removeNote n)}}
```

Comment:
```suggestion
                @action={{fn this.removeNote n}}
```

Reply:
yup, there's https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-unnecessary-curly-parens.md for that

I'll enable that in the upcoming lint-configs release 😃 (the one that will enable a bunch of new rules)